web/src/components/dashboard/PSRDocumentationLink.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Box, Text } from 'rebass';
+
+const PSRDocumentationLink: React.FC = () => {
+  return (
+    <Box>
+      <Text>
+        {/* eslint-disable-next-line */}
+        Git documentation for Platform Services Registry and a link to report issues can be found <Link to="https://github.com/bcgov/platform-services-registry">here</Link>
```

Comment:
I think we need a period at the end of the sentence. Also, why do we want to disable eslint for this line? Can we use variables for this URL? btw, I think it would be nice to create a new tab for this.

Reply:
I disabled eslint because it wanted me to either have the <Link> element on a new line, and threw an error if I had a whitespace after "found", thus I could not have the link appear as "found here", and we were stuck with "foundhere".

Reply:
There's a way to add white space in HTML format.

Reply:
Oh, right. I forgot about that. I'll fix this.

Reply:
Please revert this pr and send a new pr with correct linting.